torch/nn/quantized/modules/linear.py

```diff
@@ -0,0 +1,63 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+import torch
+from ...modules.linear import Linear as NNLinear
+from ...._jit_internal import weak_module
+
+@weak_module
+class Linear(NNLinear):
+    r"""
+    A module that wraps the quantized fbgemm linear operator function
```

Comment:
I'd just say that it implements quantized linear with quantized inputs/outputs. Fbgemm can be mentioned as an implementation detail